### DISCUSSION TITLE
Não consigo programar... 😓😓

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#6340] [General]
Baixei recentemente o NetBeans 18 e o JDK 20.0.2, porém não consigo digitar nada no meu código... pois só me aparece a opção "History" no projeto, não aparece a opção "Source"!!
Sou estudante e iniciante em programação Java, alguém poderia me ajudar com isso??

![Captura de tela 06](https://github.com/apache/netbeans/assets/142186401/aab151e0-b097-465c-859e-02a853476825)

**Comment by mbien** [comment]
close NetBeans, remove the config folder and start again. Don't import any plugins or config when you start NetBeans again.

config folder location: https://github.com/apache/netbeans#log-config-and-cache-locations

**Reply by ghost** [reply]
Você teria alguma vídeo de tutorial para fazer isto??
Não entendi muito bem como fazer isto... não sei muito sobre programação. 😅😅
